cpa/profiling/profile_gmm.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
 
-def _compute_mixture_probabilities((cache_dir, normalization_name, 
-                                    preprocess_file, images, gmm, meanvector, 
-                                    loadings)):
-    import numpy as np        
+from __future__ import absolute_import
+def _compute_mixture_probabilities(
```

Comment:
Ugh, why even tuplize these arguments :man_facepalming: Thanks for catching this. I'd honestly just name this variable `args`, rather than the concatenated name.